### PR TITLE
feat(frontend): Add run ID and workflow name in RunDetails config. Part of #4921

### PR DESCRIPTION
**Pull request by elikatsis**
**Description of your changes:**

Extend RunDetails config table with the run ID and the underlying workflow name.

Part of #4921.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: frontend/src/pages/RunDetails.tsx [hunk 1]
```diff
@@ -857,6 +857,8 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
     return !workflow.status
       ? []
       : [
+          ['Run ID', runMetadata ? runMetadata.id : '-'],
```
Comment (Bobgy) [inline frontend/src/pages/RunDetails.tsx:860]: nit: maybe?
```suggestion
          ['Run ID', runMetadata?.id || '-'],
```

File: frontend/src/pages/RunDetails.tsx [hunk 2]
```diff
@@ -857,6 +857,8 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
...
+          ['Workflow name', workflow.metadata?.name ? workflow.metadata.name : '-'],
```
Comment (Bobgy) [inline frontend/src/pages/RunDetails.tsx:861]:
```suggestion
          ['Workflow name', workflow.metadata?.name || '-'],
```

**Review by Bobgy — COMMENTED** [review]
/lgtm
Thank you for the contribution!

**Comment by Bobgy** [comment]
Can you include a screenshot for the change?

**Comment by elikatsis** [comment]
>Can you include a screenshot for the change?

Sure! Sorry for not attaching one from start.

Here it is. The two first rows of the Run Details table are added by this PR: `Run ID` and `Workflow name`:
image

File: frontend/src/pages/RunDetails.tsx (hunk 1 above)
Comment (elikatsis) [reply frontend/src/pages/RunDetails.tsx]: Done!

File: frontend/src/pages/RunDetails.tsx (hunk 2 above)
Comment (elikatsis) [reply frontend/src/pages/RunDetails.tsx]: Done!

**Comment by Bobgy** [comment]
Looks neat! : )
/lgtm
/approve

Note, renamed PR name to `feat` type, because it's not fix to a bug

**Pull request merged**